conjur-oss/README.md

```diff
@@ -299,6 +299,7 @@ The following table lists the configurable parameters of the Conjur OSS chart an
 |`conjurLabels`|Extra Kubernetes labels to apply to Conjur resources|`{}`|
 |`database.url`|PostgreSQL connection string. If left blank, an integrated PostgreSQL deployment is created.|`""`|
 |`database.password`|PostgreSQL database password string. Unused if an external Postgres database is configured. See [PostgreSQL Database Password Restrictions](#postgresql-database-password-restrictions) below.|`""`|
+|`database.remotePassword`|PostgreSQL database password string. Unused if an external Postgres database is configured. See [PostgreSQL Database Password Restrictions](#postgresql-database-password-restrictions) below.|`""`|
```

Comment:
> external Postgres database is configured
Is the external DB limited to Postgres? Is it going to work with external Postgres DB or only with Aurora "wrapper"?

Reply:
@oburstein-hub , can you provide some high-level context as to how the remotePassword will be used?

Without this context, I'm finding this comment a bit confusing, and the comment doesn't seem to match the code that you've added below.

For example, based on this comment and the preceding comment, *if an external Postgres database is not configured* , then you have two passwords that can come into play:
- `database.password`, which is auto-generated
- `database.remotePassword`, which is optional
Are these two passwords intended to be mutually exclusive? If so, then there's an issue as @uCatu points out, since whenever `database.remotePassword` is defined, the `database.password` will also be defined via auto-generation.

And if you look at the code below, the `database.remotePassword` isn't necessarily "Unused if an external Postgres database is configured"... i.e. it's possible to configure both `database.url` and `database.remotePassword` at the same time. I don't know if this is intentional or not.

I guess what I'm asking boils down to this: There are 3 chart values of interest here:
- database.url
- database.password
- datbase.remotePassword
Are these all intended to be mutually exclusive? If not, what combinations are valid, and when would you use those combinations?

We should probably make the choices clearer in the README.md.